framework/files/system/oem/99_elemental-register.yaml

```diff
@@ -2,4 +2,4 @@ name: "Elemental operator bootstrap"
 stages:
   network.after:
     - commands:
-        - systemctl start elemental-register.service
+        - systemctl start elemental-register.timer
```

Comment:
I think we can consider entirely removing this file and start the registration timer by default. The default enabled services are listed in a systemd Elemental branding package in [OBS](https://build.opensuse.org/package/show/isv:Rancher:Elemental:Dev/systemd-presets-branding-Elemental).

Reply:
I would leave it here right now for simplicity, this issue already has three PRs as dependencies. 
I can open a follow up issue to discuss this if you think it's the case.

Reply:
Yes, in fact. Also thinking it further probably we can keep it like this as running the service from the cloud-config allows us to set a different start or setup for different scenarios (live media, active, fallback or recovery systems).

We could omit it in live media and in that case run something more aggressive like `elemental-register --install` (assuming we have a flag to trigger installation or no)